settings-docker.yaml

```diff
@@ -6,7 +6,7 @@ llm:
   mode: ${PGPT_MODE:mock}
 
 embedding:
-  mode: ${PGPT_MODE:sagemaker}
+  mode: ${PGPT_EMBEDDING_MODE:sagemaker}
```

Comment:
Could you separate this change into another PR? We need to discuss it what it is the possible impact in current implementations